test/integration/nighthawk_test_server.py

```diff
@@ -40,6 +40,29 @@ def _substitute_yaml_values(runfiles_instance, obj, params):
   return obj
 
 
+# A list of message pieces that should be ignored even if logged by the test
+# server at a warning or an error severity.
+#
+# Other messages logged at either of these severities fail the test.
+_TEST_SERVER_WARN_ERROR_IGNORE_LIST = frozenset([
+    # TODO(#580) Remove these once our use of Envoy's API is updated to v3.
+    "Configuration does not parse cleanly as v3",
+    "Deprecated field: type envoy.api.v2.Cluster",
+    "Deprecated field: type envoy.api.v2.listener.Filter",
+    "Deprecated field: type envoy.config.filter.network.http_connection_manager.v2.HttpFilter",
+
+    # TODO(mum4k): Identify these and file issues or add explanation as necessary.
+    "Unable to use runtime singleton for feature envoy.http.headermap.lazy_map_min_size",
```

Comment:
not actionable, but I think I remember issues being filed for this, it might be a bug in Envoy.

Reply:
Thank you, if you do know the issue number, please let me know. Also your help in identifying background for the remaining ignored warnings would be much appreciated.

Reply:
Note - we will be addressing some of the "Using deprecated extension name ..." warnings in my next PR. I will leave this in here as is for now, but leave the TODO and come back after the next PR gets merged in to see if additional cleanup is needed. Tracking in #582.